README.md

```diff
@@ -138,14 +138,26 @@ Paths provided are relative to the current directory from which you execute the
 
 There are plenty of examples you can use to start off:
 
-- `example`: contains an example app with version 2 of the specification. It will give you an idea how to annotate your comments in order to include them in the output specification.
+- `examples/js`: contains an example app with version 2 of the specification. It will give you an idea how to annotate your comments in order to include them in the output specification.
+- `examples/ts`: contains an example TypeScript app with version 2 of the specification. 
 - `test/cli`: CLI tests you can read to get ideas about the available functionalities of the CLI. (apart from the obvious help menu)
 - `test/example`: various assets for tests you can also re-use for starting definitions, routes, etc.
 
 ### CLI
 
 On top of the Node API, there is also a [command line interface](./docs/CLI.md).
 
+
+### Using with TypeScript at Runtime
+
+Yes, you can use swagger-jsdoc with your TypeScript project. There are some adjustments in the configuration necessary, though.
+1. You can install types via `@types/swagger-jsdoc`.
+2. Typescript is usually transpiled to JavaScript. So in order to generate the Swagger document the `apis: [...]` option has to point to the transpiled JavaScript files, and not to the TypeScript source files. In default configuration, the transpiled files are located in the `dist` directory (and the files end with `.js`).
```

Comment:
As discussed in https://github.com/Surnet/swagger-jsdoc/issues/194, the important message here is that swagger-jsdoc is reading any type of source file parsing for comments no matter which version of input files an implementer will configure for the `apis`.

The cases for the different approaches should be more concise. "usually transpiled" may be unhelpful for developers in a hurry for a solution how to use swagger-jsdoc without even knowing what is transpilation. The situation in which transpilation and "dist" specifics hit the face of a newby introduced to a TS project with low JS knowledge will throw the newby in pain. 

Suggestion:

It's important to know note that `swagger-jsdoc` reads any type of source code and scans for comments containing `@swagger` annotations. The way `swagger-jsdoc` is integrated in a project based on Typescript depends on the way how the OpenAPI/Swagger information is managed through project's code.

If the information regarding OpenAPI specification is to be extracted before the Typescript code reaches production, then run `swagger-jsdoc` on raw source code and double-check for references and links between entities in the specification to ensure that something developed locally is not referenced on production.

The information can also be extracted through CI/CD systems which work with already transformed Typescript code and references and links related to actual information from production. In this case, please double-check that glob patters in your swagger definitions are covering all the code and all the cases which are relevant to be shared on production. Please also verify that Typescript configuration in your project does not strip down comments from the code you scan for specification annotations, as comments are what `swagger-jsdoc` reads for compiling your specification.

It is possible to have a combination of raw/transformed/transpiled code coming from various folders within your project. It very much depends on your project's workflows and Typescript configurations.